runtime/databricks/automl_runtime/forecast/pmdarima/training.py

```diff
@@ -65,25 +66,51 @@ def fit(self, df: pd.DataFrame) -> pd.DataFrame:
         # Impute missing time steps
         history_pd = self._fill_missing_time_steps(history_pd, self._frequency_unit)
 
+        history_timedelta = history_pd['ds'].max() - history_pd['ds'].min()
+        timedeltas = history_pd['ds'].diff()
+        min_timedelta = timedeltas.iloc[timedeltas.values.nonzero()[0]].min()
```

Comment:
Good catch. I'll remove `timedeltas` and `min_timedelta`. I originally implemented another check for `min_timedelta`, but I realized that it is not necessary, because we do `_fill_missing_time_steps`